projects/core/src/user/occ/user.service.spec.ts

```diff
@@ -76,6 +76,24 @@ describe('OccUserService', () => {
     });
   });
 
+  describe('update user details', () => {
+    it('should update user details for the given username', () => {
+      const userUpdates: User = {
+        title: 'mr',
+      };
+      service.updateUserDetails(username, userUpdates).subscribe(_ => _);
+
+      const mockReq = httpMock.expectOne(req => {
+        return req.method === 'PATCH' && req.url === endpoint + `/${username}`;
```

Comment:
Let's also validate that the new user details are in the request body, or rather that we get the expected request body for a given set of user details.

Having the 3 fields we expose in the form would be nice too.

Reply:
Are you referring to this? https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/1873/files/3a714095a99cb20833e08ab3cdb8a272c3494d19#diff-5697e5d8eaa2e2e3dd18de4e952de5f8R92

Reply:
Yes, that's what I'm referring to : )